magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -53,20 +52,6 @@ MAGIC:
         use: true
         factor: 1.143
 
-    increase_nsb_m1:
```

Comment:
it seems that you used an older config file and now are trying to merge changes that counteract previous changes